athenz/consts.go

```diff
@@ -1,15 +1,17 @@
 package athenz
 
 const (
-	AUDIT_REF            = "done by terraform provider"
-	ROLE_SEPARATOR       = ":role."
-	GROUP_SEPARATOR      = ":group."
-	POLICY_SEPARATOR     = ":policy."
-	RESOURCE_SEPARATOR   = ":"
-	SERVICE_SEPARATOR    = "."
-	SUB_DOMAIN_SEPARATOR = "."
-	PREFIX_USER_DOMAIN   = "home."
-	EXPIRATION_LAYOUT    = "2006-01-02 15:04:05"
-	MEMBER_EXPIRATION    = "member expiration"
-	EXPIRATION_PATTERN   = "[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9] [0-9][0-9]:[0-9][0-9]:[0-9][0-9]"
+	AUDIT_REF               = "done by terraform provider"
+	ROLE_SEPARATOR          = ":role."
+	GROUP_SEPARATOR         = ":group."
+	POLICY_SEPARATOR        = ":policy."
+	RESOURCE_SEPARATOR      = ":"
+	SERVICE_SEPARATOR       = "."
+	SUB_DOMAIN_SEPARATOR    = "."
+	PREFIX_USER_DOMAIN      = "home."
+	EXPIRATION_LAYOUT       = "2006-01-02 15:04:05"
+	MEMBER_EXPIRATION       = "member expiration"
+	EXPIRATION_PATTERN      = "[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9] [0-9][0-9]:[0-9][0-9]:[0-9][0-9]"
+	MEMBER_REVIEW_REMINDER  = "member review reminder"
+	REVIEW_REMINDER_PATTERN = "[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9] [0-9][0-9]:[0-9][0-9]:[0-9][0-9]"
```

Comment:
use the same constant and rename it to date pattern maybe?

Reply:
Done, I renamed it